fasttext/debian/libfasttext.install

```diff
@@ -0,0 +1 @@
+usr/lib/lib*.so*
```

Comment:
Not multiarch ready?

Reply:
I've fixed to make it multiarch ready, but it needs to patch for a moment (I'll feedback to upstream)
how about apply patch?

```
diff --git a/dev/tasks/linux-packages/package-task.rb b/dev/tasks/linux-packages/package-task.rb
index f9173ac82..0e47f4cc6 100644
--- a/dev/tasks/linux-packages/package-task.rb
+++ b/dev/tasks/linux-packages/package-task.rb
@@ -196,6 +196,10 @@ RELEASE=#{@rpm_release}
         File.open(spec, "w") do |spec_file|
           spec_file.print(spec_data)
         end
+        patch_files = Dir.glob("#{yum_dir}/*.patch")
+        unless patch_files.empty?
+          cp(patch_files, tmp_dir)
+        end
 
         cd(yum_dir) do
           threads = []
diff --git a/dev/tasks/linux-packages/yum/build.sh b/dev/tasks/linux-packages/yum/build.sh
index 1ada8782a..d7168a652 100755
--- a/dev/tasks/linux-packages/yum/build.sh
+++ b/dev/tasks/linux-packages/yum/build.sh
@@ -84,6 +84,9 @@ if [ -n "${SOURCE_ARCHIVE}" ]; then
       ;;
     *)
       run cp /host/tmp/${SOURCE_ARCHIVE} rpmbuild/SOURCES/
+      if [ ! "`yum/*.patch`" >/dev/null ]; then
+        run cp -f /host/tmp/*.patch rpmbuild/SOURCES/
+      fi
       ;;
   esac
 else
```

and how to apply in Rakefile(define_yum_build_sh_task)?

Reply:
Can we pass `-DCMAKE_INSTALL_LIBDIR=...` option to `cmake`?

Reply:
Thanks, already done in the previous commit.
https://github.com/groonga/packages.groonga.org/pull/17/files#diff-3d48126cda2cd47b692ce3958d988341R50

It requires patch to release, so I'm doubt about how to copy patch Rakefile(to yum/*) -> package task  (to yum/tmp/) ->  build.sh (tmpdir to SOURCES) in good manner.

Reply:
I understand.
The patch isn't acceptable because Apache Arrow package doesn't need it.
I'll add the feature later with other changes.
Please don't apply the patches for fastText to RPM package.

Reply:
Thanks, I'll drop patch.